src/sentry/search/events/base.py

```diff
@@ -120,6 +122,10 @@ def _resolve_user_display_alias(self, _: str) -> SelectType:
         columns = ["user.email", "user.username", "user.ip"]
         return Function("coalesce", [self.column(column) for column in columns], USER_DISPLAY_ALIAS)
 
+    def _resolve_transaction_status(self, _: str) -> SelectType:
+        column = self.column(TRANSACTION_STATUS_ALIAS)
+        return Function("coalesce", [column], TRANSACTION_STATUS_ALIAS)
```

Comment:
I'm not sure we need a [coalesce](https://clickhouse.tech/docs/en/sql-reference/functions/functions-for-nulls/#coalesce) for transaction status since its only a single column